apple/DemoApp/Demo/MuteButtonView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MuteButton.swift
+//  Ferrostar Demo
+//
+//  Created by Marek Sabol on 08/10/2024.
+//
+
```

Comment:
1. The mute button isn't specific to the demo app but is reusable, so it should move to the `FerrostarSwiftUI` module. All similar controls are under FerrostarSwiftUI > Views > Controls.
2. We remove the automatic Xcode header comments.